# Reproduced from externalpackages/m1qn3/patch/m1qn3.f.patch
# in ISSM Subversion trunk.
--- src/src/m1qn3.f     2009-10-20 06:39:35.000000000 -0400
+++ m1qn3.f     2021-08-13 14:44:30.276019165 -0400
@@ -802,7 +802,7 @@
      &        "  iter  simul  stepsize            f                |g|",
      &        "       |g|/|g0|"
           write(io,
-     &        '(1x,i5,2x,i5,2x,1pd8.2,2x,d21.14,2x,d11.5,2x,d10.4)')
+     &        '(1x,i5,2x,i5,2x,1pd9.2,2x,d21.14,2x,d12.5,2x,d11.4)')
      &        niter, isim, t, f, gnorm, eps1
       endif
       if (impres.ge.5) write (io,940) eps1
